app/gimlet/Cargo.toml

```diff
@@ -5,12 +5,14 @@ name = "gimlet"
 version = "0.1.0"
 
 [features]
+traptrace = []
 
 [dependencies]
 cortex-m = { workspace = true }
 cortex-m-rt ={ workspace = true }
 cfg-if = { workspace = true }
 stm32h7 = { workspace = true, features = ["rt", "stm32h753"] }
+ringbuf = { path = "../../lib/ringbuf"  }
```

Comment:
We should make this `optional = true`, then specify it above with
```toml
[features]
traptrace = ["ringbuf"]
```

Reply:
Good call -- fixed!